testsuite/features/secondary/min_salt_install_package_and_patch.feature

```diff
@@ -3,17 +3,26 @@
 
 @skip_if_github_validation
 @scope_salt
-Feature: Install a patch on the client via Salt through the UI
+Feature: Install a package and a patch on the SUSE client via Salt through the UI
```

Comment:
Nice to adapt the feature title to what the feature exactly does.

Alternatively, we could have cut into two features: one for installing, and one for patching. I am not sure this would have been a good idea though. It would be clearer yes, but wouldn't that be extra complications? You choose.